### PR TITLE
Feat/lw 8771 upgrade local network to conway

### PR DESCRIPTION
**Pull request by AngelCastilloB**
# Context

A working _local-network_ is mandatory!
Current `cardano-node` stack version for Conway era is not fully stable.

# Proposed Solution

Updated _local-network_ configurations and init scripts to restore itself (what's possible with current `cardano-node` stack version).

Everything not yet supported is commented out or skipped; please refer to: LW-9182 , LW-9183 and LW-9184 .

### CODE REVIEW

File: packages/e2e/local-network/scripts/make-babbage.sh [hunk 1]
```diff
@@ -153,14 +156,15 @@ mkdir -p "${ROOT}/genesis/shelley"
...
+mv "${ROOT}/genesis.conway.json" "${ROOT}/genesis/shelley/genesis.conway.json"
 mv "${ROOT}/genesis.json" "${ROOT}/genesis/shelley/genesis.json"
 
 jq --raw-output ".protocolConsts.protocolMagic = ${NETWORK_MAGIC}" "${ROOT}/genesis/byron/genesis-wrong.json" >"${ROOT}/genesis/byron/genesis.json"
 
 rm "${ROOT}/genesis/byron/genesis-wrong.json"
 
 $SED -i "${ROOT}/genesis/shelley/genesis.json" \
-  -e 's/"slotLength": 1/"slotLength": 0.2/' \
+  -e 's/"slotLength": 1/"slotLength": 0.1/' \
```
[the remark this answers is not in the document]
Comment (mkazlauskas) [reply packages/e2e/local-network/scripts/make-babbage.sh:167]: I remember this used to not work. Don't remember the exact issues we ran into (probably something with rewards), but the network didn't work properly.

[the remark this answers is not in the document]
Comment (iccicci) [reply packages/e2e/local-network/scripts/make-babbage.sh]: High speed configuration was restored, so this was reverted. This should be solved

**Review by mirceahasegan — APPROVED** [review]
Fantastic work, @iccicci 🥇 🚀

**Review by mkazlauskas — APPROVED** [review]
Great work guys! :muscle: :rocket:

**Pull request merged**